client/battle/BattleFieldController.h

```diff
@@ -25,6 +25,18 @@ class Canvas;
 class IImage;
 class BattleInterface;
 
+enum ERelativePosition

```

Comment:
Identical to BattleHex::Direction?

Reply:
Zombie code I missed on my own review :(. Deleted!